docs/synapse/devguides/devops_cell.rst

```diff
@@ -27,6 +27,18 @@ Cell implementations can extend the configuration variables available by specify
 
 Depending on deployment requirements, a combination of methods can be used for loading the configurations into the Cell.
 
+.. note::
+    The service directory (refered to as ``dirn``) should be considered a persistent directory for a given Synapse
+    service. Inside of this directory there are several files stored which are neccesary in order for a given instance
```

Comment:
using the word "service" is problematic because it is confusing with Storm service. Can't we just call it a "cell directory"? Perhaps eliminate use of "service" through this whole directory.

This is what happens when we use common words as proper nouns.

Reply:
replaced service verbiage with cell verbiage.